content/pt-br/_collections/navigations/header.md

```diff
@@ -0,0 +1,102 @@
+---
+links:
+  - title: 'Descubra'
+    items:
+      - title: 'Vitrines'
+        subtitle: 'Seleção de website construído com Nuxt'
+        slug: 'showcases'
+        to: '/showcases'
+        icon: 'showcases.svg'
+        color: 'bg-sand'
+      - title: 'Estudos de caso'
+        subtitle: 'Como as companhias usam Nuxt'
+        slug: 'case-studies'
+        to: '/case-studies'
+        icon: 'case-studies.svg'
+        color: 'bg-sand-dark'
+      - title: 'Depoimentos'
+        subtitle: 'O que eles pensam sobre nós'
+        slug: 'testimonials'
+        to: '/testimonials'
+        icon: 'testimonials.svg'
+        color: 'bg-sand-darker'
+  - title: 'Aprenda'
+    items:
+      - title: 'Documentação'
+        subtitle: 'Crie sites rápido facilmente'
+        slug: 'docs'
+        to: '/docs'
+        icon: 'docs.svg'
+        color: 'bg-green-500'
+      - title: 'Exemplos'
+        subtitle: 'Entenda tudo no Nuxt'
+        slug: 'examples'
+        to: '/examples'
+        icon: 'examples.svg'
+        color: 'bg-green-600'
+      - title: 'Tutoriais'
+        subtitle: 'Aprenda com casos práticos'
+        slug: 'tutorials'
+        to: '/tutorials'
+        icon: 'tutorials.svg'
+        color: 'bg-green-700'
+      - title: 'Cursos de Mestres'
+        subtitle: 'Aprenda com os especialistas'
+        href: 'https://masteringnuxt.com/?utm_source=nuxt&utm_medium=link&utm_campaign=nsite'
+        icon: 'master-courses.svg'
+        color: 'bg-green-800'
+  - title: 'Explore'
+    items:
+      - title: 'Deploy'
+        subtitle: 'Como implementar o Nuxt'
+        slug: 'deployments'
+        to: '/deployments'
+        icon: 'deployments.svg'
+        color: 'bg-indigo-light'
+      - title: 'Módulos'
+        subtitle: 'Estender o poder do Nuxt'
+        href: 'https://modules.nuxtjs.org'
+        icon: 'modules.svg'
+        color: 'bg-indigo'
+      - title: 'Temas'
+        subtitle: 'Introdução aos temas'
+        slug: 'themes'
+        to: '/themes'
+        icon: 'themes.svg'
+        color: 'bg-indigo-dark'
+      - title: 'Cursos em Vídeos'
+        subtitle: 'Aprenda passo-a-passo'
+        slug: 'video-courses'
+        to: '/video-courses'
+        icon: 'video-courses.svg'
+        color: 'bg-indigo-darker'
+  - title: 'Comunidade'
+    items:
+      - title: 'Anúncios'
+        subtitle: 'Últimas notícias sobre Nuxt'
+        slug: 'announcements'
+        to: '/announcements'
+        icon: 'announcements.svg'
+        color: 'bg-mint-lighter'
+      - title: 'Times'
```

Comment:
`Equipas`?

Reply:
`Equipas` is from pt-pt. `Times` is practically the same thing as `Equipes`, I changed it to `Equipes` because its looks better.

Reply:
The `Equipas` is from pt-pt. `Times` is pretty much the same thing as `Equipes`, but when referring to `Development Team` it is usually `Time`, `Equipes` is not used much on this occasion.